helper-cli/src/main/kotlin/commands/GenerateScopeExcludesCommand.kt

```diff
@@ -112,6 +112,13 @@ private fun getScopeExcludesForPackageManager(packageManagerName: String): List<
                 comment = "Development dependencies."
             )
         )
+        "GoMod" -> listOf(
+            ScopeExclude(
+                name = "all",
+                reason = ScopeExcludeReason.BUILD_TOOL_OF,
+                comment = "Scope dependencies used to build all targets including non-released artifacts like tests.."
```

Comment:
I would drop the initial "Scope" from the sentence for consistency with the other excludes. And one dot at the end is sufficient :)

Reply:
Done.

Reply:
Done.